tests/schedulers/test_scheduler_dpm_multi_inverse.py

```diff
@@ -54,6 +54,7 @@ def check_over_configs(self, time_step=0, **config):
 
             output, new_output = sample, sample
             for t in range(time_step, time_step + scheduler.config.solver_order + 1):
+                t = scheduler.timesteps[t]
```

Comment:
We used `0`,`1`,`2` as `timestep` index here, but they are not even in the `self.timestepes`.  In the previous implementation, it will default to the last timesteps when it is outside of the timesteps range, but I don't think it is intended. I changed it here, I think it makes more sense this way. Let me know if it's not the case